src/components/WhaleSightingViewer/WhaleSightingFilters.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../Constants.scss";
+
+.filterCard {
+    background-color: $background;
+}
+
+.btn-search-btn {
+    background-color: $button-color;
+    color: $dark-font-color;
+    margin: 1%;
+}
+
+.selectOption {
+    background-color: $input-field-color;
+    border: none;
+    margin: 0.5%;
+    border-radius: 5px;
+    overflow: auto;
+    width: 45%;
+}
+
+input {
+    background-color: $input-field-color;
+    border: none;
+    color: $dark-font-color;
+    margin: 0.5%;
+    border-radius: 5px;
+    width: 45%;
+}
```

Comment:
can we give this a class name? I'm wary of styling html elements like this as it could apply site-wide